### ISSUE TITLE
pq: syntax error at or near "master"

### ISSUE THREAD

**Issue by enugentdt** [concourse/concourse#3547]
## Bug Report

Upon attempting to PUT into a keyval resource, I receive the following error from Concourse itself, and _not_ from the keyval-resource:

```
Mar 19 22:06:05 web1.concourse.stm.inf.demilletech.net concourse[4057]: {"timestamp":"2019-03-20T02:06:05.684043661Z","level":"error","source":"atc","message":"atc.pipelines.scheduler.tick.scheduling.try-start-next-pending-build.failed-to-save-output","data":{"build-id":2024,"build-name":"51","error":"pq: syntax error at or near \"master\"","pipeline":"isoscribe-frontend","resource":"keyval-beta","resource-type":"keyval","session":"18.4.34813.3.7","step":"keyval-beta","team":"isoscribe","version":{"GIT_AUTHOR":"<uname>","GIT_ID":"<git commit>","GIT_MSG":"\"Merge branch 'master' of https://<gitlab repo url>\"","UPDATED":"Wednesday, 20-Mar-19 02:06:05 UTC","UUID":"f11a3df1-35e4-40fb-b396-8f8578e2c0f1","VERSION":"1553047424"}}}

Mar 19 22:06:05 web1.concourse.stm.inf.demilletech.net concourse[4057]: {"timestamp":"2019-03-20T02:06:05.684846995Z","level":"info","source":"atc","message":"atc.pipelines.scheduler.tick.scheduling.try-start-next-pending-build.errored","data":{"build-id":2024,"build-name":"51","error":"pq: syntax error at or near \"master\"","pipeline":"isoscribe-frontend","session":"18.4.34813.3.7","team":"isoscribe"}}
```

## Steps to Reproduce

It appears that a simple PUT into keyval with quotes in the variable causes it?

## Expected Results

Not this issue

## Actual Results

This issue

## Additional Context

This happened immediately after updating to v5.0.0. Running on Debian 9

## Version Info

* Concourse version: v5.0.0
* Deployment type (BOSH/Docker/binary): Binary
* Infrastructure/IaaS: VMware
* Browser (if applicable): Chrome, but probably not applicable (I hope)
* Did this used to work? Yes

**Comment by archon21** [comment]
Yeah I am getting same error as well

**Comment by xtremerui** [comment]
Hi what is the postgres version ? Have you tried update it to latest version?

**Comment by enugentdt** [comment]
I currently have 9.6.11 running. Should I update to 11?

**Comment by vito** [comment]
@enugentdt Nah, 9.6.11 is fine - this looks like a bug in our code for saving versions. I'll put it at the top of our backlog.

**Comment by vito** [comment]
The fix for this is available in v5.0.1: https://concourse-ci.org/download.html#v501

It's actually a security issue, so anyone affected should upgrade ASAP.
